packages/light-apps/src/stores/onboardingStore.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017-2018 @polkadot/light-apps authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { action, observable } from 'mobx';
+import store from 'store';
```

Comment:
Can we use localForage instead? I've tried both a lot, I think localForage is superior:
- it's async
- has Observable support
- has more types of storage